.circleci/config.yml

```diff
@@ -120,6 +120,7 @@ commands:
       - run: sudo apt-get update
       - run: sudo apt-get install python tcl
       - run: sudo apt-get install python3-venv
+      - run: sudo apt-get install python3-pip
```

Comment:
Do you know if we need to include this in our `building` docs: https://github.com/mozilla/application-services/blob/main/docs/building.md?

Reply:
We shouldn't, this just isn't installed by default with python in the CI environment. Typical local building you would get pip along with python3 installation (or a venv)